packages/liquidator/index.js

```diff
@@ -21,6 +21,14 @@ const {
 const { getAbi, getAddress } = require("@uma/core");
 const { getWeb3 } = require("@uma/common");
 
+const SUPPORTED_CONTRACT_VERSIONS = [
```

Comment:
Do you think that other classes or even dApps would re-use some of these contract versions? Perhaps we should put the list of all possible contract versions in a `common` file and allow each deriving class to specify what versions it supports.

Reply:
yup! good idea. We can pull these out in a separate PR I think. My next step is to try and generalize a fair amount of the specific changes done in this PR.

Reply:
Ok sgtm to leave to the next PR but I do think that these hardcoded dicts should be refactored to another file